db/seeds.rb

```diff
@@ -211,6 +218,24 @@
   approved: true,
   user_id: User.first.id,
 )
+
+Rails.logger.info "10. Creating Badges"
+
+Badge.create!(
+  title: Faker::Lorem.word,
+  description: Faker::Lorem.sentence,
+  badge_image: File.open(Rails.root.join("app", "assets", "images", "#{rand(1..40)}.png")),
+)
+
+Rails.logger.info "10. Creating FeedbackMessages"
```

Comment:
I think it should be `11. Creating FeedbackMessages` here.

Reply:
@just1602 you could submit a PR if you feel like to :-)

1. Go to https://github.com/thepracticaldev/dev.to/blob/master/db/seeds.rb
1. Click on the edit icon:

![Screenshot 2019-04-30 at 9 57 35 AM](https://user-images.githubusercontent.com/146201/56947781-752c0e80-6b2e-11e9-9afc-2e8315d83ab8.png)

1. Go to line 230 and make the change 🎉

Reply:
@rhymes thats probably the smallest PR I ever made #2631